css/style.css

```diff
@@ -15,5 +16,212 @@ body {
+    font-weight: 550;
+    font-size: 23px;
+    text-transform: uppercase;
+    text-decoration: none;
+    padding: 10px;
+} */
+#meet_karma{
+color: black;
+
+}
+#meet_karma:hover{
+    color: green;
+}
+.intro{
+    width: 100%;
+    background-image: url("../img/first-background.jpg");
+    background-size:cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100vh;
+    color: white;
+    margin-bottom: 50px;
+}
+.everyone{
+    padding: 50px;
+}
+/* <!--text style--> */
+.intro h1{
+    font-weight: 100;
+    font-size:70px;                                                                                              px;
+    margin-top: 15rem;
+}
+.intro h2{
+    font-weight:50;
+    font-size: 40px;                                                                                               px;
+    margin-bottom: 50px;
+}
+.intro h3{
+    font-weight:50;
+    font-size: 40px;                                                                                               px;
+    margin-bottom: 15rem;
+}
+.everyone h3{
+    text-align:center;
+    font-size: 65px;
+    font-weight: 40;
+    color:rgb(46, 48, 51);
+}
+.everyone p{
+    color: black;
+    font-size: 25px;
+}
+/* button */
+button{
+    background-color: orangered;
+    color: white;
+    padding: 15px 30px;
+    border: none;
+    border-radius: 4px;
+}
+.tree_item{
+    display: flex;
+    justify-content:space-around;
+    margin-bottom: 50px;
+}
+.item{
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    font-size: 1.5rem;
+    text-align: center;
+    padding: 50px;
+    margin: 0 20px;
+    /* width: 10rem;
+    height: 10rem; */
+}
+.social_media{
+    width: 20px;
+    height: 20px;
+    padding: 15px;
+    border: 1px solid lightgray;
+    border-radius: 40%;
+}
+.icon{
+    width: 10rem;
+    height: 10rem;
+}
+.extention-container{
+    display: flex;
+    margin-top:2rem;
+    height: 439px;
+}
+.extention-image1{
+    background-color: rgba(224, 99, 58, 0.1);
+    width: 60%;
+    text-align: center;
+}
+.extention-image1 p{
+    margin-top: 10rem;
+    font-size: x-large;
+    font-style: italic;
+}
+.extention-image1 button{
+    background-color: orangered;
+    color: white;
+    padding: 15px 5px;
+    border-radius: 5px;
+}
+.extention-image1 button a{
+    color: white;
+    text-decoration: none;
+}
+.footer{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.column{
+    display: flex;
+    flex-direction: column;
+}
+.wrapper-form{
+    width: 50%;
+}
+.image{
+    background-image: url(../img/store-image_by-andrew-neel-unsplash.jpg);
+    background-size: cover;
+    background-position: right bottom;
+    width: 50%;
+}
+main{
+    width: 100vw;
+    display: flex;
+}
+.row{
+    display: flex;
+    justify-content: space-between;
+}
+input{
+    margin-bottom: 15px;
+    padding: 5px;
+}
+form{
+    padding:60px;
+}
+.name{
+    width: 48%;
```

Comment:
These numbers looks puzzling as they are very precise. Any specific reason why this is 48?

Reply:
No, it's not a specific reason.